src/shogun/machine/IterativeMachine.h

```diff
@@ -57,7 +59,8 @@ namespace shogun
 		virtual bool continue_train()
 		{
 			this->reset_computation_variables();
-
```

Comment:
let me understand this again. Is `continue_train` called from `train`? I.e. can we make sure the continure features are set on training start?

Reply:
Or where exactly are those set?

Reply:
yes it is called from within train_machine

Reply:
ok thanks!